remove_ok_only_comments.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
Please remove this file from commit.
We don't need it, one migration complete, this script be useless.

Reply:
so can it be done like i execute the script , which replaces all of the ok comments with blank lines and then commit after removing the script ?

Reply:
Yes.
Just not as script to commit.
Keep it only to change files.